Makefile

```diff
@@ -100,3 +100,7 @@ sidecar: sidecar-wp-cli
 sidecar-wp-cli:
 	docker build -t $(SIDECAR_WP_CLI_IMG) -f $(SIDECAR_DIR)/wp-cli.dockerfile $(SIDECAR_DIR)
 	docker push $(SIDECAR_WP_CLI_IMG)
+
+sidecar-mysql:
+	docker build -t $(SIDECAR_MYSQL_CLI_IMG) -f scripts/sidecar-mysql/Dockerfile scripts/sidecar-mysql/
```

Comment:
SIDECAR_MYSQL_CLI_IMG is not declared in the make file
"sidecar-mysql" is not added to the sidecar target list